frontend/src/pages/Home.js

```diff
@@ -94,78 +94,80 @@ const Home = () => {
             {/* WORKOUTS CONTAINER */}
             <div className="workouts" style={{ marginBottom: "5vh" }}>
               {/* BEGINNER WORKOUTS */}
-              {workouts &&
-                workouts.map((e) => {
-                  return (
-                    <>
-                      <h1 style={{ margin: "2vh 0 2vh 7%" }}>{e.title}</h1>
-                      <div className="card-container">
-                        <Tilt
-                          glareEnable={true}
-                          glarePosition={"all"}
-                          className="card"
-                        >
-                          <Link to="/workout">
-                            {/* <img src={e.image} alt="" /> */}
-                            <h2>{e.workout1.title}</h2>
-                            <div className="rating">
-                              <AiFillStar className="star" color="#ff385c" />
-                              <AiFillStar className="star" />
-                              <AiFillStar className="star" />
-                            </div>
-                          </Link>
-                        </Tilt>
-                      </div>
-                    </>
-                  );
-                })}
-
+              {/* {data && */}
+              <h1 style={{ margin: "2vh 0 2vh 7%" }}>Beginner Workouts</h1>
```

Comment:
Create an separate component for Workouts, Component s are same for Beginner, Intermediate and Advanced workouts, just pass the workouts object as param to the newly component.